tests/example-success/expected_results.json

```diff
@@ -1,74 +1,167 @@
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 9: forceful question",
+       "test_code": "Test 9: forceful question",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 10: shouting numbers",
+       "test_code": "Test 10: shouting numbers",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 11: no letters",
+       "test_code": "Test 11: no letters",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 12: question with no letters",
+       "test_code": "Test 12: question with no letters",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 13: shouting with special characters",
+       "test_code": "Test 13: shouting with special characters",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 14: shouting with no exclamation mark",
+       "test_code": "Test 14: shouting with no exclamation mark",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 15: statement containing question mark",
+       "test_code": "Test 15: statement containing question mark",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 16: non-letters with question",
+       "test_code": "Test 16: non-letters with question",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 17: prattling on",
+       "test_code": "Test 17: prattling on",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 18: silence",
+       "test_code": "Test 18: silence",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 19: prolonged silence",
+       "test_code": "Test 19: prolonged silence",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 20: multiple line question",
+       "test_code": "Test 20: multiple line question",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 21: starting with whitespace",
+       "test_code": "Test 21: starting with whitespace",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 22: ending with whitespace",
+       "test_code": "Test 22: ending with whitespace",
+       "status"   : "pass",
+       "message"  : null
+     }
+     ,
+     {
+       "name"     : "Test 23: non-question ending with whitespace",
+       "test_code": "Test 23: non-question ending with whitespace",
+       "status"   : "pass",
+       "message"  : null
+     }
    ],
    "version": 2,
-   "status": "pass"
+   "message": "Test summary: 0 of 23 tests failed",
```

Comment:
As per [the docs](https://github.com/exercism/docs/blob/main/building/tooling/test-runners/interface.md#message):

> When the `status` is not `error`, either set the value to `null` or omit the key entirely.

In this case, the `status` is `pass`, so the key should be omitted or its value set to `null`.

Reply:
Added PR https://github.com/exercism/fortran/issues/151 for this